tests/tests_files_elasticsearch_certs_incomplete.yml

```diff
@@ -21,8 +21,8 @@
         - "{{ __test_cert }}"
         - "{{ __test_key }}"
 
-    - block:
-      - name: deploy config to send to elasticsearch
+    - name: deploy config to send to elasticsearch
+      block:
```

Comment:
This does not look correct. You essentially deleted the list item with the `include_role` below.